### PR TITLE
Show an error to the users if their job fails

### LINKED ISSUE #560: Runtime error should not be logged in Honeybadger
**Describe the bug**
This `RuntimeError` https://github.com/sul-dlss/pre-assembly/blob/master/app/lib/pre_assembly/digital_object.rb#L91 is being caught by Honeybadger.

It seems this is a data error and should just be presented to the user and not go into HB.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Presently, it just raises an error and the user never sees their job complete.
Fixes #560

## Was the documentation (README, DevOpsDocs, wiki, consul, etc.) updated?
n/a

### CODE REVIEW

File: config/initializers/okcomputer.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@
 OkComputer::Registry.register 'feature-resque-down', OkComputer::ResqueDownCheck.new
 
 # check for backed up resque queues: this is a low volume app, so the threshold is low
-Resque.queues.each do |queue|
-  OkComputer::Registry.register "feature-#{queue}-queue-depth", OkComputer::ResqueBackedUpCheck.new(queue, 5)
-end
+# Resque.queues.each do |queue|
```
Comment (ndushay) [inline config/initializers/okcomputer.rb]: why is this being removed?  is it useless?

Comment (jcoyne) [reply config/initializers/okcomputer.rb]: This was just commented out for testing (because it hits redis). I've restore this.

**Comment by peetucket** [comment]
This could potentially obviate the need for #595 and we could close that (unless there are other important exceptions we are raising that we'd rather log and proceed on instead).

**Comment by peetucket** [comment]
Looks like a rebase is needed :(

**Comment by mjgiarlo** [comment]
@peetucket I resolved the conflicts.

**Comment by ndushay** [comment]
> This could potentially obviate the need for #595 and we could close that (unless there are other important exceptions we are raising that we'd rather log and proceed on instead).

wrong PR/issue number?

**Comment by peetucket** [comment]
> > This could potentially obviate the need for #595 and we could close that (unless there are other important exceptions we are raising that we'd rather log and proceed on instead).
> 
> wrong PR/issue number?

Oops, yes, sorry, should be #533 [WIP PR]

**Pull request merged**